cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -478,7 +478,7 @@ func (m *gceManagerImpl) getCpuAndMemoryForMachineType(machineType string, zone
 		}
 		m.cache.AddMachineToCache(machineType, zone, machine)
 	}
-	return machine.GuestCpus, machine.MemoryMb * bytesPerMB, nil
+	return machine.GuestCpus, machine.MemoryMb * 1024 * 1024, nil
```

Comment:
Why not use units.MiB in this file as well, since it's already being added?

Reply:
done